testing/katalon/Object Repository/Contacts/createIndividualContact/addAnotherEmailAddressLink.rs

```diff
@@ -0,0 +1,103 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@stairaku 
Not sure if it matters but I can show you how to substantially shorten the size of these selectors. I can book some time with you this upcoming sprint to show you

Reply:
for sure! and also I don't know how to assign these PRs to myself and add labels. That would also be helpful. Thanks!

Reply:
Has this been addressed yet?

Reply:
Not yet. I'm coordinating with Jonny to check this out

Reply:
21 days now! can we merge this?

Reply:
yes please! Sorry about the delay